doc/tech/master.md

```diff
@@ -0,0 +1,99 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Common NFVI Telco Taskforce
+
+## Table of Contents
+* [1. Overview](#1.0)
+* [2. Scope](#2.0)
+* [3. Principles](#3.0)
+* [4. Technology Direction](#4.0)
+  * [4.1 SR-IOV](#4.1)
+  * [4.2 PCI-Pass Through](#4.2)
+  * [4.3 SmartNICs](#4.3)
+  * [4.4 Acceleration Cards](#4.4)
+  * [4.5 FPGA](#4.5)
+  * [4.6 GPUs/NPUs](#4.6)
+  * [4.7 EPA/NFD](#4.7)
+* [5. Transitional Plans](#5.0)
+* [6. Roadmap](#6.0)
+* [7. Available Specifications](#7.0)
+
+<a name="1.0"></a>
+## 1. Overview
+
+Initially organized early in 2019, the Common Network Function Virtualisation Infrastructure Telecom Taskforce (CNTT) was initially created in response to rapid changes in how networking applications are being designed, built and managed, plus a growing recognition of a perceived functional gap between the previous standard infrastructure models and the architectures needed to support Network Function Virtualization (NFV) applications.  Organizationally the Common Telco Network Function Virtualisation Infrastructure (NFVI)  project, jointly hosted by GSMA and the Linux Foundation, operates as an open committee responsible for creating and documenting an industry aligned Common NFVI Framework.  The CNTT group was created with the intent that it would create the NFVI framework, and eventually morph into an on-going project under the auspices of the GSMA and the Linux Foundation umbrellas.  The final on-going operational form of the group will be determined as the project evolves. 
+
+<a name="2.0"></a>
+## 2. Scope
+
+Within the framework of the Common Telecom NFVI vision, there are three levels of documents needed to document the components and allow the practical application of the systems. They are, as highlighted in **Figure 1**:  **Reference Model**, **Reference Architecture**, and **Reference Implementation**.
+
+<p align="center"><img src="./figures/tech_scope_doc_types.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 1:</b> Scope of CNTT</p>
+
+<a name="2.1"></a>
+## 2.1 Specification Types
+
+- **Reference Model**: focuses on the __**NFVI Abstraction**__ and how NFVI services and resources are exposed to VNFs. It written at a high enough level that as new **Reference Architectures** and **Reference Implementations** are added, the model document should require few or no changes. Additionally, the Reference Model is intended to be neutral towards VMs or Containers.
```

Comment:
I know there is a caveat at the end, but I would suggest changing "NFVI" where used in this paragraph to something more agnostic that allows for containerisation, physical (bare metal) infrastructure as well as virtualised infrastructure. Same with VNFs.  Maybe something like the following, and remove the last sentence (or expand to include bare metal):

> focuses on the __**Infrastructure Abstraction**__ and how infrastructure services and resources are exposed to Network Functions.

Reply:
@tomkivlin As you know ETSI defines:
Virtualised Network Function (VNF): implementation of an NF that can be deployed on a Network Function Virtualisation Infrastructure (NFVI).

Network Functions Virtualisation Infrastructure (NFVI): totality of all hardware and software components that build up the environment in which VNFs are deployed.

Our glossary uses the same terminology and from that perspective do you think the use of NFVI is OK?

Reply:
@pgoyal01 I think using NFVI is fine when discussing *virtualisation* and *virtualised* network functions (i.e. those that run in VMs).  Cloud native / containerised network functions may or may be virtualised (i.e. may run on bare metal) and thinking more long term, I think there is value in the Reference Model being applicable to physical network functions, where there is no virtualisation or containerisation, but the concepts of standard interfaces etc. will still bring value through standardisation and efficient certification of the NF software.  

As such, in this paragraph, I don't think NFVI is appropriate as it only covers one of those three scenarios.

Reply:
@tomkivlin fixed in: https://github.com/cntt-n/CNTT/pull/731/commits/0c7db9e73c6295efb710fa79354b1a4bf0fd1c01